frontend/src/app/modules/attachments/attachment-list/attachment-list.component.ts

```diff
@@ -33,6 +33,7 @@ import {filter} from "rxjs/operators";
 import {States} from "core-components/states.service";
 import {AngularTrackingHelpers} from "core-components/angular/tracking-functions";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import { HttpClient } from '@angular/common/http';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/ac011261452f67db461044df50f218f9e6a7f7f2/frontend/src/app/modules/attachments/attachment-list/attachment-list.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>